boards/qorvo/decawave_dwm3001cdk/doc/index.rst

```diff
@@ -0,0 +1,65 @@
+.. _decawave_dwm3001cdk:
+
+Decawave DWM3001CDK
+#################
+
+Overview
+********
+
+The DWM3001CDK development board includes the DWM3001C module, battery
+connector and charging circuit, LEDs, buttons, Raspberry-Pi and USB
```

Comment:
this development board comes with a free raspberry pi?

Reply:
rpi and usb connector, that wording is from https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/qorvo/decawave_dwm1001_dev/doc/index.rst?plain=1#L10
Pretty confusing now that you point it out though
I'll change it here at least